src/snowcli/cli/snowpark/function/commands.py

```diff
@@ -201,25 +150,22 @@ def function_update(
             log.info(
                 "Return type or handler types do not match. Replacing the function."
             )
-            replace = True
+            create_or_replace = True
```

Comment:
WDYT about  encapsulating this into helper method?

Reply:
Good idea, done